.github/workflows/workflow.yml

```diff
@@ -35,8 +74,8 @@ jobs:
         run: tox -e bandit
       - name: Safety Check
         run: tox -e safety
-      - name: License Check
-        run: tox -e liccheck
+      #- name: License Check
+      #  run: tox -e liccheck
```

Comment:
Why has this been commented?

Reply:
@marcofavorito  for testing needs, cause liccheck fails on the new version of packaging module

Reply:
are we going to reintroduce it at some point? Maybe we can merge this PR and open an issue to keep track of it

Reply:
I think @DavidMinarsch solved this problem in another PR